random cheroot related test suite failures with Python 3.13

**I'm submitting a ...**
- [X] bug report
- [ ] feature request
- [ ] question about the decisions made in the repository

**What is the current behavior?**

Similarly to https://github.com/cherrypy/cheroot/issues/734, I am seeing several of the following test failures with Python 3.13:
```
E                   pytest.PytestUnraisableExceptionWarning: Exception ignored in: <function IOBase.__del__ at 0x7fd73c938ae0>
E
E                   Traceback (most recent call last):
E                     File "/usr/lib/python3.13/_pyio.py", line 418, in __del__
E                       self.close()
E                       ~~~~~~~~~~^^
E                     File "/usr/lib/python3.13/_pyio.py", line 1313, in close
E                       self.flush()
E                       ~~~~~~~~~~^^
E                     File "/usr/lib/python3.13/_pyio.py", line 1274, in flush
E                       self._flush_unlocked()
E                       ~~~~~~~~~~~~~~~~~~~~^^
E                     File "/usr/lib/python3.13/vendor-packages/cheroot/makefile.py", line 33, in _flush_unlocked
E                       n = self.raw.write(bytes(self._write_buf))
E                     File "/usr/lib/python3.13/socket.py", line 737, in write
E                       return self._sock.send(b)
E                              ~~~~~~~~~~~~~~~^^^
E                   OSError: [Errno 9] Bad file number
```
Based on the trace, this seems like the same cheroot issue (likely because `helper.CPWebCase` inherits from `webtest.WebCase` imported from cheroot). I think these errors should be ignored in CherryPy as well to make the test suite pass.

**To Reproduce:**
Run the test suite with Python 3.13.0.

**Environment:**

- Cheroot version: 10.0.0
- CherryPy version: 18.10.0
- Python version: 3.13.0
- OS: Oracle Solaris (but this doesn't seem to be platform dependent)

--- a/pytest.ini
+++ b/pytest.ini
@@ -63,6 +63,10 @@ filterwarnings =
   # TODO: connection properly.
   ignore:unclosed <socket.socket fd=:ResourceWarning
 
+  # Python 3.13 no longer ignores IOBase errors raised by the close(),
+  # which exposed a possible race condition in cheroot test class.
+  ignore:Exception ignored in. <function IOBase.__del__:pytest.PytestUnraisableExceptionWarning
+
 junit_duration_report = call
 junit_family = xunit2
 junit_suite_name = cherrypy_test_suite

